### ISSUE TITLE
Is there a loop in the command?

### ISSUE THREAD

**Issue by mangojuice123** [kevwan/tproxy#11]
I'm new to Docker and I'm confused when I try to understand the command

`docker run --rm -it -p <listen-port>:<listen-port> -p <remote-port>:<remote-port> kevinwan/tproxy:v1 tproxy -l 0.0.0.0 -p <listen-port> -r host.docker.internal:<remote-port>`

I'm gonna split the command into 2 parts:

1. `docker run --rm -it -p <listen-port>:<listen-port> -p <remote-port>:<remote-port> kevinwan/tproxy:v1`: The flag `-p` map 2 ports of the host to the container, that't alright;
2. `tproxy -l 0.0.0.0 -p <listen-port> -r host.docker.internal:<remote-port>`: The `-r host.docker.internal:<remote-port>` part cause the container access the host port directly, which I think may cause a loop like this:

![Docker 端口映射](https://user-images.githubusercontent.com/57697266/192939797-84df069f-bdb2-4baa-aabb-8eb4900f1240.png)

Could you please help me to figure it out?

**Comment by kevwan** [comment]
Yes, I think so. It's actually the same port that the traffic goes to.
